runtime/lua/vim/lsp/util.lua

```diff
@@ -1956,18 +1961,19 @@ local function make_position_param(window, offset_encoding)
 end
 
 --- Creates a `TextDocumentPositionParams` object for the current buffer and cursor position.
----
+---@param row integer|nil 1-indexed row number of position, defaults to current line
+---@param col integer|nil 0-indexed column number of position, defaults to current character
 ---@param window integer|nil: window handle or 0 for current, defaults to current
 ---@param offset_encoding string|nil utf-8|utf-16|utf-32|nil defaults to `offset_encoding` of first client of buffer of `window`
 ---@return table `TextDocumentPositionParams` object
 ---@see https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocumentPositionParams
-function M.make_position_params(window, offset_encoding)
+function M.make_position_params(row, col, window, offset_encoding)
```

Comment:
We should not break the function by changing the signature.

Reply:
d'oh, I forget the util module is public... I got annoyed by repeatedly passing `(nil, nil, row, col)`, though I suppose I could get past that by just passing a table as args